packages/Form/summary/src/Summary.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import Alert from '@axa-fr/react-toolkit-alert';
+
+const defaultClassName = 'af-alert';
+const defaultClassModifier = 'danger';
+const defaultProps = {
+  title: 'Invalid form',
+  messages: null as string,
+  isVisible: false,
+  className: defaultClassName,
+  classModifier: defaultClassModifier,
+};
+
+interface MessagesProps {
+  messages: string[];
+}
+
+const Messages: (props: MessagesProps) => JSX.Element = ({ messages }: MessagesProps) => {
+  if (!messages || !messages.length) {
```

Comment:
Not needed. We know that we have an array with messages since the component is not exposed.